apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -147,22 +181,37 @@ const Template = ({
         });
     }
 
+    function renderLeadingOrTrailingSlot (slotName: 'leading' | 'trailing', slotValue: typeof slotOptions[number]) {
+        if (slotValue === slotOptions[0]) {
```

Comment:
You could place these inside of an object instead of the `if` statement and assign the html markup to each key (there would only be two) then at the end return the slot required or nothing. Would simplify this method imo.

e.g

slot = {
[slotOne]: html`<markup>`
[slotTwo]: html`<markup>`
}

reutnr slot[slotValue] || nothing;